tools/src/wyvern/tools/lexing/WyvernLexer.x

```diff
@@ -253,7 +253,7 @@ import wyvern.tools.errors.ToolError;
     terminal Token cSquareBracket_t ::= /\]/ {: RESULT = token(RBRACK,lexeme); :};
 
  	terminal Token shortString_t ::= /(('([^'\n]|\\.|\\O[0-7])*')|("([^"\n]|\\.|\\O[0-7])*"))|(('([^']|\\.)*')|("([^"]|\\.)*"))/ {:
- 		RESULT = token(STRING_LITERAL,lexeme.substring(1,lexeme.length()-1));
+ 		RESULT = token(STRING_LITERAL,lexeme.substring(1,lexeme.length()-1).replace("\\n", "\n"));
```

Comment:
One thing we can do is to process the string character by character translating the escape sequences \t, \b, \n, \r, \f, \', \", \\ into the appropriate characters as we encounter them. Does this sound okay?